<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QWidget" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnTest">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>190</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Testinappi</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCard2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassword2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEvents">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>470</y>
     <width>311</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tapahtumat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBalance">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>380</y>
     <width>311</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Saldo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDeposit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>311</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Talletus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnWithdraw">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>311</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Otto</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCard_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Valitse</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAccType">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAccountID">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSuljeOptions">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>580</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sulje</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
